generate.py

```diff
@@ -67,8 +77,15 @@ def generate(src="src", target="output", format="png", style="modern-blue"):
 
 
 def main():
+    parser = OptionParser()
+    parser.add_option("-r", "--rebuild", dest="rebuild", default=False,
+                      help="Rebuild exist output image")
+
```

Comment:
This looks good. It's really a good practice to introduce option parser. Would you mind one more change?
 I think it may be much better if you can add `format` and `style`  option as well so that the users can set this options  on command line to output  custom format.